graylog2-server/src/test/java/org/graylog2/decorators/FormatStringDecoratorTest.java

```diff
@@ -34,13 +35,14 @@
 import static org.graylog2.rest.models.messages.responses.ResultMessageSummary.create;
 
 public class FormatStringDecoratorTest {
+    private final Engine templateEngine = Engine.createDefaultEngine();
 
     @Test
     public void testFormat() {
 
         final DecoratorImpl decorator = getDecoratorConfig("${field_a}: ${field_b}", "message", true);
 
-        final FormatStringDecorator formatStringDecorator = new FormatStringDecorator(decorator);
+        final FormatStringDecorator formatStringDecorator = new FormatStringDecorator(decorator, templateEngine);
```

Comment:
Why is it not possible to mock out the engine instead of supplying a real instance?